grammars/gfm.cson

```diff
@@ -145,7 +145,9 @@
   }
   {
     'begin': '\\A---$'
-    'end': '^---$'
+    # Added '...' as an acceptable end-block 
```

Comment:
Would you mind removing the comments? The `git blame` on this file will trace back to this PR which will tell the story of why this is here.